main/src/main/scala/org/clulab/numeric/mentions/package.scala

```diff
@@ -609,6 +609,36 @@ package object mentions {
       throw new RuntimeException(s"Error: cannot convert mention of type [${m.getClass.toString}] to DateMention!")
   }
 
+  def toDateMentionHoliday(mention: Mention): DateMention = mention match {
+    case m: DateMention => m
+
+    case m: RelationMention =>
+      val holiday = getArgWords("holiday", m)
+      if(holiday.isEmpty)
+        throw new RuntimeException(s"ERROR: could not find argument holiday in mention ${m.raw.mkString(" ")}!")
+
+      val year = getArgWords("year", m) match {
+        case Some(y) => Some(y)
+        case _ => None
+      }
```

Comment:
This looks suspicious.  Isn't it the same as `val year = getArgWords("year", m)`, unless one might be concerned that `null` is returned.  It doesn't look like it can be.

Reply:
Yes, checking this again is not necessary.